lib/MusicBrainz/Server/EditSearch/Predicate/VoteCount.pm

```diff
@@ -0,0 +1,56 @@
+package MusicBrainz::Server::EditSearch::Predicate::VoteCount;
+use Moose;
+use namespace::autoclean;
+
+use MusicBrainz::Server::Constants qw( :vote );
+use MusicBrainz::Server::Types qw( VoteOption );
+use MusicBrainz::Server::Validation qw( is_integer );
+
+extends 'MusicBrainz::Server::EditSearch::Predicate::ID';
+
+around operator_cardinality_map => sub {
+    my ($orig, $self) = splice(@_, 0, 2);
+    my %orig_map = $self->$orig;
+    return map { $_ => 1 + $orig_map{$_} } keys %orig_map;
```

Comment:
Is that because the vote type is now `args.0`? Can't we just make that `args.2` and leave the rest alone? Shouldn't matter where it appears in the actual HTML. (Maybe I'm completely misunderstanding why this is needed.)